whois-api/src/test/java/net/ripe/db/whois/api/elasticsearch/ElasticSearchTestInstance.java

```diff
@@ -0,0 +1,47 @@
+package net.ripe.db.whois.api.elasticsearch;
+
+
+import net.ripe.db.whois.common.profiles.WhoisProfile;
+import org.apache.http.HttpHost;
+import org.elasticsearch.client.RestClient;
+import org.elasticsearch.client.RestClientBuilder;
+import org.elasticsearch.client.RestHighLevelClient;
+import org.slf4j.Logger;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Profile;
+import org.springframework.stereotype.Component;
+
+import javax.annotation.Nullable;
+import java.util.List;
+
+import static org.slf4j.LoggerFactory.getLogger;
+
+@Profile({WhoisProfile.TEST})
+@Component
+public class ElasticSearchTestInstance implements ElasticRestHighlevelClient {
+
+    private static final Logger LOGGER = getLogger(ElasticIndexService.class);
+    private final RestHighLevelClient client;
+
+    @Autowired
+    public ElasticSearchTestInstance(@Value("#{'${elastic.host:}'.split(',')}") final List<String> elasticHosts) {
+        this.client = getEsClient(elasticHosts);
+    }
+
+    @Nullable
+    private RestHighLevelClient getEsClient(final List<String> elasticHosts) {
```

Comment:
Can you name the class according to what it does? It looks like this is the "non auth" instance vs. `ElasticSearchInstance` which requires authentication.

Reply:
May be ElasticSearchTestNonAuthInstance? It is used only in test and it is in test package so I named it as ElasticSearchTestInstance

Reply:
Renamed to ElasticSearchNonAuthInstance

Reply:
Thanks @maggarwal13 I think this name is clearer as it describes what it does (but still stays in `src/test/`)